atlasdb-plugin/src/main/groovy/com/palantir/atlasdb/AtlasPlugin.groovy

```diff
@@ -0,0 +1,34 @@
+package com.palantir.atlasdb
+
+import com.palantir.gradle.javadist.AtlasCreateStartScriptsTask
+import com.palantir.gradle.javadist.DistTarTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPlugin
+
+class AtlasPlugin implements Plugin<Project> {
+    @Override
+    void apply(Project project) {
+        project.plugins.apply(JavaPlugin)
+        AtlasPluginExtension ext = project.extensions.create('atlasdb', AtlasPluginExtension)
+
+        AtlasCreateStartScriptsTask createScript = project.tasks.create('createAtlasScripts', AtlasCreateStartScriptsTask)
+
+        // If the java-distribution plugin is applied make the DistTarTask build the script
+        project.tasks.withType(DistTarTask)[0]?.dependsOn createScript
+
+        // TODO: figure out how to get people to bring in the dep
+        // probably should force them to specify it themselves...
```

Comment:
Nah I think if applying the gradle atlas plugin means you expect to get the "cli" then we should just force the dependency to be applied by the plugin if possible, because they're not going to pull the cli jars in via the atlas-client dependency.